test/run.test.js

```diff
@@ -366,7 +366,7 @@ test('run should accept a unix socket/windows pipe', (t) => {
 })
 
 for (let i = 1; i <= 5; i++) {
-  test(`run should count all ${i}xx status codes`, (t) => {
+  test(`run should count all ${i}xx status codes`, { skip: true }, (t) => {
```

Comment:
Has to disable this temporarily as it fails consistently only on node 15.6.0.
We may need to investigate this separately.

Reply:
Can you please make it skip only on node v15?

Reply:
Done